### ISSUE TITLE
Failure when connecting to certain Metamask accounts

### ISSUE THREAD

**Issue by Moliholy** [AugurProject/augur-ui#2314]
Good morning,

I recently started using Augur and after the very first time I bought shares I was no longer able to use my ethereum account with Metamask. Whenever I try to link it I see the following error in the javascript console:

```
Uncaught Error: Actions must be plain objects. Use custom middleware for async actions.
    at p (createStore.js:152)
    at store.js:23
    at index.js:14
    at dispatch (applyMiddleware.js:35)
    at load-account-data-from-local-storage.js:19
    at Array.map (<anonymous>)
    at load-account-data-from-local-storage.js:19
    at index.js:11
    at dispatch (applyMiddleware.js:35)
    at load-account-data.js:15
```

I've been also debugging and `action` seems to be undefined.

The account I used is the following: https://etherscan.io/address/0x1b6ba2a0f34a8f0b375fc1f8f6e9d5fe3520ce5d

It always shows I have 0 ETH (which is false) and 0 REP (which is also false), and whenever I try to access the account tab I see the following:

![screen shot 2018-09-23 at 13 07 31](https://user-images.githubusercontent.com/2722756/45927304-a8237980-bf31-11e8-8bc1-9dc3b2d775f0.png)

Any idea?

### EDIT

I've also detected notifications from my last interaction are still showing up every ~5 seconds, even after I've deleted them. It might be related to that [last transaction](https://etherscan.io/tx/0x71c35d2771de841831e227c955e31c1cd176911b98052bba119ffee99a59b7db)?

**Comment by adrake33** [comment]
Added to internal tracking system at https://app.clubhouse.io/augur/story/16854/eth-and-rep-balance-show-zero-after-changing-mm-accounts

**Comment by adrake33** [comment]
@Moliholy This is being caused by an error that occurs with pulling user data from localStorage. We have a fix coming out in next release.

**Comment by Moliholy** [comment]
Awesome! Thanks a lot!

**Comment by adrake33** [comment]
Closing this issue, as it should now be resolved in version 1.6.1.
